src/components/admin/form-components/withFalcorSearch.jsx

```diff
@@ -125,6 +113,6 @@ export const withSearchableConcept = (
     extraPathSets: [[]],
   };
 
-  Searchable.displayName = `SearchableField(${getDisplayName(WrappedField)})`;
+  Searchable.displayName = `FalcorSearch(${getDisplayName(WrappedField)})`;
```

Comment:
I think it's simpler just naming it the same as the HOC no? Just do `withFalcorSearch`

Reply:
Yep, I can do that. I just copied Will's way of doing the validated fields, as he omitted the `with` too :)

Reply:
Ahh okay, fair enough, I think it's slightly clearer just naming it the same as the HOC though

Reply:
I added `with`!